diamond.c

```diff
@@ -0,0 +1,23 @@
+//Bettina Benitez
```

Comment:
Great job! I appreciate that you included the user input feature - very nice touch. The only suggestion I have is to add more comments!